packages/cli/cli/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.39.14] - 2024-08-21
+
+- Fix: Format validation is enforced on `date` fields that are specified in examples in fern definitions.
```

Comment:
this will technically be true for OpenAPI as well (since we go OpenAPI -> FD)

Reply:
updated